src/Tasks/dependency-check-build-task/dependency-check-build-task.ts

```diff
@@ -193,6 +254,18 @@ async function run() {
     console.log("Ending Dependency Check...");
 }
 
+function logDebug(message: string) {
```

Comment:
Debug message function. Will print out debug messages, but only if generic build pipeline variable for debug messages "system.debug" is set to true.